QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private IConsole _messageConsole;
+        private readonly IList<IMessageListener> _listeners = new List<IMessageListener>();
+
+        public MessageEmitter( IConsole messageConsole) {
+            _messageConsole = messageConsole;
+        }
+        public void Start() {
+            string lineInput = "";
+
+            while (lineInput != "/exitQuestionBot") {
+                lineInput = _messageConsole.ReadLine();
+
+                if ( lineInput != "/exitQuestionBot" ) {
```

Comment:
I somehow forgot that we deal with messages that don't start with "/question" already - so the user wouldn't see anything. (we just return null)